### ISSUE TITLE
Latest version doesn't install (Looking for wrong name)

### ISSUE THREAD

**Issue by crabcrabcam** [deadhead420/arch-linux-anywhere#196]
Hey, just found out about this yesterday and installed to USB today. It installed fine using SUSEImageWriter but when I plugged it in, after the boot it said `looking for device at /devices/sortBy/label` or something like that, and the name of the device it was looking for was `ARCH_201702`. However the USB got called `ARCH_ANY226`. That's the name whether through DD or SUSEImageWriter. I'd try and fix it myself, but I looked through the code and couldn't find where that comes from. 

I also don't seem to be able to rename, but that's probably just me not having a clue!

Looks great though and I'll still try it when this gets fixed.

Thanks, Cam!

**Comment by deadhead420** [comment]
Shittttt.... This should have been fixed awhile ago... Ugh..... What did you use to create th USB? SUSEImageWriter? Did you try creating it with DD?

**Comment by crabcrabcam** [comment]
Yeah, I tried DD after. I also formatted the USB to just "stick" as the name so it could pick whatever. It chose that `ARCH_ANY226` one every time.

**Comment by Daylien** [comment]
Yep same here. But after that, i tried it with rufus on windows and now it works great.

**Comment by crabcrabcam** [comment]
@Daylien, HAHA, that's great! The one that shouldn't work works fine! I guess I'll have to install Windows to install Linux. And I just wiped it off with Feliz (That one has way more problems... It seems to work but it then forgets to actually install the OS!)

I'll give that a go and report back.

**Comment by crabcrabcam** [comment]
Oh for gods sake. I just reinstalled it through RUFUS and now it looks for `ARCH_ANY226`. I specifically called it `ARCH_201702` as well... I guess I'll give it another go and let RUFUS set the name...

**Comment by Daylien** [comment]
That should do it. And for the case it doesn't work try [MultiBootUSB](http://multibootusb.org/).

**Comment by crabcrabcam** [comment]
Yep. RUFUS works! That's about all the info I can give you now. This should work. Sorry I couldn't be more help and actually fix the issue...

**Comment by deadhead420** [comment]
I will look into it a bit more later on, don't have time right this second... This was an issue for awhile until I finally came up with a fix which actually involves doing a search and replace on the binary data. It should have been fixed for awhile now but I guess not.... Just to verify... I'm assuming you're using UEFI correct?

**Comment by crabcrabcam** [comment]
@deadhead420, Yeah. Using UEFI. Should I be using legacy?

**Comment by deadhead420** [comment]
Legacy would work without any issues but UEFI should work as well... Going to make a USB with dd when I get home and see if I can recreate the issue. I'll get to the bottom of it and report back.

**Comment by crabcrabcam** [comment]
The system I was installing off is Antergos Arch to the latest update. DD ran through ZSH with oh-my-zsh if that matters. But this seems to have worked and is installing, however the website says "NVidia closed source drivers" whilst the installer says "Open source drivers". Also the "Install guide" page is broken. Sorry to raise so many issues I can't fix, if there's any other way I can help then I'd be happy to give it a shot :)

**Comment by deadhead420** [comment]
Doesn't really matter what shell you run dd thought. I also use zsh + oh-my-zsh, love it :) The installer has both open source and closed source nvidia drivers, you just have to select which ones you want, I'll update that on the site. As for the install guide I just haven't gotten around to doing it yet, haven't had the time. Let me know how the install goes.

**Comment by deadhead420** [comment]
FIXED. Thanks again for pointing this out. Was a very easy and obvious fix. The way I replace the label for EFI is to use a utility called xxd which is a hex editor to search and replace the binary data for "ARCH_201702" with "ARCH_ANY226". This is a very complex solution but it works. What happened was since the size of the binary data changes with each release of the archiso some of the binary for "ARCH_201702" was moved onto another column so sed does not find all of it and the search and replace is not successful (in the previous release the binary data for "ARCH_201701" was all on one column so the search and replace was successful). The fix was very simple and only required me to look right at the top of the man page for xxd (man pages save lives). The default for xxd is to only show 16 octets per line, using the -c 256 parameter I increased this to 256 octets per line so the binary data for "ARCH_201701" is not split into two lines and the search and replace is a success. Tested and worked perfect all I had to do was add "-c 256" to my xxd command. Gotta love such a simple fix. Thanks again for pointing this out to me. Will be fixed in the next release on March 1st.
